cli/src/commands/gerrit.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright 2024 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt::Debug;
+
+use clap::Subcommand;
+use indexmap::IndexSet;
+use jj_lib::commit::Commit;
+use jj_lib::repo::Repo;
+
+use crate::cli_util::{
+    get_git_repo, resolve_multiple_nonempty_revsets, user_error, CommandError, CommandHelper,
+    RevisionArg,
+};
+use crate::ui::Ui;
+
+/// Interact with Gerrit Code Review.
+#[derive(Subcommand, Clone, Debug)]
+pub enum GerritCommand {
+    /// "Mail in" a revision to Gerrit; this is the primary way to create
+    /// changes or update changes for code review.
+    ///
+    /// This command modifies each commit in the revset to include a `Change-Id`
+    /// footer in its commit message if one does not already exist. This ID is
+    /// NOT compatible with jj IDs, and is a Gerrit-specific ID.
+    ///
+    /// Note: this command takes a revset as an argument, and will mail in all
+    /// revisions in the revset to Gerrit appropriately, so you may post trees
+    /// or ranges of commits to Gerrit for review.
+    ///
+    /// Note: this command does not actually send an email or use a Mail
+    /// Transfer Agent, but rather uploads the revision to Gerrit for review
+    /// through underlying Git logic. The term "mail" is used to disambiguate
+    /// from the word "submit", which is what Gerrit calls "merging an approved
+    /// change into a target branch", and is used to refer to the concept of
+    /// "mailing in" something to be read by someone else.
+    Mail(MailCommand),
+}
+
+pub fn cmd_gerrit(
+    ui: &mut Ui,
+    command: &CommandHelper,
+    subcommand: &GerritCommand,
+) -> Result<(), CommandError> {
+    match subcommand {
+        GerritCommand::Mail(review) => {
+            cmd_mail(ui, command, review)?;
+        }
+    }
+    Ok(())
+}
+
+#[derive(clap::Args, Clone, Debug)]
+pub struct MailCommand {
+    /// The revset, selecting which commits are mailed in to Gerrit. This can be
+    /// any arbitrary set of commits; they will be modified to include a
+    /// `Change-Id` footer if one does not already exist, and then mailed off to
+    /// Gerrit for review.
+    #[arg(long, short = 'r')]
+    revision: Vec<RevisionArg>,
+
+    /// The origin where your changes will be sent. This MUST be a reference of
```

Comment:
I was going to say that we should replace "origin" by "remote", but that's not quite right either. How about "The remote branch to your changes are meant for." or something like that (feel free to reword)?

Reply:
I made it "The location where your changes are intended to land." Does that make sense?